app/Services/Attendance/GeneralWorkSchedulePublisher.php

```
@@ -130,27 +162,156 @@ private function publishLocked(
             'profile_id' => $profile->id,
             'payroll_policy_key' => WorkScheduleProfilePublication::DURATION_FIRST_V2,
             'effective_from' => $effectiveFrom->toDateString(),
             'effective_to' => null,
+            'definition_hash' => $lockedPlan['definition_hash'],
+            'request_key' => $lockedPlan['request_key'],
+            'payload_hash' => $lockedPlan['payload_hash'],
+            'reason' => $reason,
+            'published_by' => $actor->id,
+        ]);
+
+        return ['publication' => $publication, 'profile' => $profile];
```

PHPStan (2.2.7)

[error] Method App\Services\Attendance\GeneralWorkSchedulePublisher::writePublicationLocked() should return array{publication: App\Models\WorkScheduleProfilePublication, profile: App\Models\WorkScheduleProfile|null} but returns array{publication: Illuminate\Database\Eloquent\Model, profile: App\Models\WorkScheduleProfile}.
Offset 'publication' (App\Models\WorkScheduleProfilePublication) does not accept type Illuminate\Database\Eloquent\Model.

(return.type)